src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/Switch.cs

```diff
@@ -23,7 +23,7 @@ public abstract class Switch
         private int _switchSetting;
         private volatile bool _initialized;
         private bool _initializing;
-        private volatile string _switchValueString = string.Empty;
+        private volatile string? _switchValueString = string.Empty;
```

Comment:
Just double checking, does this really need to be nullable given that it is set to a default value (string.Empty)?

Reply:
Yes since it may be set to `null` later.